i18n/index.js

```diff
@@ -1,3 +1,5 @@
-export * as en from './en.json';
-export * as de from './de.json';
-export * as es from './es.json';
+import * as en from './en.json';
+import * as de from './de.json';
+import * as es from './es.json';
+
+export { en, de, es };
```

Comment:
Any specific reason to change it back?

Reply:
Yeah, we had a hell of a time getting rollup to build it like we had it 🤦‍♂️ . This was easier.

Reply:
Ok, weird. Let's keep it simple then 😅